examples/medical-center/main.bal

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please undo this license header change

Reply:
updated in [29c72ce](https://github.com/ballerina-platform/persist-tools/pull/120/commits/29c72ce16d2edf4882433055270167e57360d0f7)